monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md

```diff
@@ -0,0 +1,129 @@
+# MONERO
+
+Monero est une cryptomonnaie sécurisée,
+confidentielle et intraçable disponible pour
+tous
+
+# QUELQUES FAITS
+
+Les utilisateurs peuvent la dépenser en
+toute sécurité, personne ne peut voir leurs
+soldes ou suivre leurs activités.
+
+Révisé le 14/11/2018 - [Version Imprimable](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_fr_PrinterFriendly.pdf)
+Créé pour la communauté par Monero Outreach
+
+[getmonero.org](https://getmonero.org/fr/)
+
+
+## Principes fondamentaux
+
+#### DÉCENTRALISATION
+
+En tant que _[projet open-source](https://github.com/monero-project/monero)_ dirigé et _[fondé](https://forum.getmonero.org/8/funding-required)_ par une _[équipe décentralisée](https://getmonero.org/fr/community/team/)_ de développeurs et membres de la communauté, il ne peut pas être censuré. La plupart des contributeurs sont des volontaires et la communauté est répartie autour du globe.
+
+#### SÉCURITÉ
+
+En tant que cryptomonnaie décentralisée, Monero est sécurisé par un grand réseau d'utilisateurs à travers le monde. Les transactions sont confirmées par un consensus distribué puis immuablement enregistrées sur la _[chaîne de blocs](https://www.mycryptopedia.com/what-is-blockchain-technology/)_.
```

Comment:
Forgot this one:
_[chaîne de blocs](https://www.mycryptopedia.com/what-is-blockchain-technology/)_
Correct link
https://getmonero.org/fr/resources/moneropedia/blockchain.html